 
 -------------- celery@ubuntu v5.4.0 (opalescent)
--- ***** ----- 
-- ******* ---- Linux-5.15.0-130-generic-x86_64-with-glibc2.35 2025-01-11 14:47:51
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         new_tdl_site:0x7f79a510ae00
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 1 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . leads.tasks.check_notifications

[2025-01-11 14:47:51,821: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-01-11 14:47:51,828: INFO/MainProcess] mingle: searching for neighbors
[2025-01-11 14:47:52,840: INFO/MainProcess] mingle: all alone
[2025-01-11 14:47:52,858: INFO/MainProcess] celery@ubuntu ready.
[2025-01-11 15:35:00,019: INFO/MainProcess] Task leads.tasks.check_notifications[dd52d238-5590-4024-854b-6c2f8b56d689] received
[2025-01-11 15:35:02,451: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[dd52d238-5590-4024-854b-6c2f8b56d689] succeeded in 2.4278742839996994s: None

worker: Warm shutdown (MainProcess)
 
 -------------- celery@ubuntu v5.4.0 (opalescent)
--- ***** ----- 
-- ******* ---- Linux-5.15.0-130-generic-x86_64-with-glibc2.35 2025-01-12 11:58:18
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         new_tdl_site:0x7f10ac90ee00
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 1 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . leads.tasks.check_notifications

[2025-01-12 11:58:18,414: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-01-12 11:58:18,420: INFO/MainProcess] mingle: searching for neighbors
[2025-01-12 11:58:19,429: INFO/MainProcess] mingle: all alone
[2025-01-12 11:58:19,445: INFO/MainProcess] celery@ubuntu ready.
[2025-01-12 12:00:00,100: INFO/MainProcess] Task leads.tasks.check_notifications[0b3cedd4-87a5-4bf3-ac5d-bb326bc54d81] received
[2025-01-12 12:00:02,846: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[0b3cedd4-87a5-4bf3-ac5d-bb326bc54d81] succeeded in 2.7419238580041565s: None
[2025-01-13 12:00:00,090: INFO/MainProcess] Task leads.tasks.check_notifications[9235bdc2-c839-4dcd-a06c-2bdf0ab4887e] received
[2025-01-13 12:00:02,880: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[9235bdc2-c839-4dcd-a06c-2bdf0ab4887e] succeeded in 2.7798270220228005s: None
[2025-01-14 12:00:00,097: INFO/MainProcess] Task leads.tasks.check_notifications[56f63802-a07e-47f6-9c38-377dfd28999d] received
[2025-01-14 12:00:03,358: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[56f63802-a07e-47f6-9c38-377dfd28999d] succeeded in 3.254934491007589s: None
[2025-01-15 12:00:00,089: INFO/MainProcess] Task leads.tasks.check_notifications[49b83f8b-9a2d-486f-8913-5919f64ff122] received
[2025-01-15 12:00:03,162: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[49b83f8b-9a2d-486f-8913-5919f64ff122] succeeded in 3.0672145009739324s: None
[2025-01-16 12:00:00,078: INFO/MainProcess] Task leads.tasks.check_notifications[ff1382e5-2799-43c2-94df-9095ac3af072] received
[2025-01-16 12:00:02,280: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[ff1382e5-2799-43c2-94df-9095ac3af072] succeeded in 2.1968150560278445s: None
[2025-01-17 12:00:00,110: INFO/MainProcess] Task leads.tasks.check_notifications[c7b1f256-d705-4f43-a827-7c05c6f9db99] received
[2025-01-17 12:00:02,520: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c7b1f256-d705-4f43-a827-7c05c6f9db99] succeeded in 2.407270575000439s: None
[2025-01-18 12:00:00,026: INFO/MainProcess] Task leads.tasks.check_notifications[4aa0c081-2f69-4e1b-a51a-797e0c4d37ad] received
[2025-01-18 12:00:02,207: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[4aa0c081-2f69-4e1b-a51a-797e0c4d37ad] succeeded in 2.1763487370917574s: None
[2025-01-19 12:00:00,109: INFO/MainProcess] Task leads.tasks.check_notifications[1cb32543-6c30-4168-bf84-dc36241e80d4] received
[2025-01-19 12:00:02,221: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[1cb32543-6c30-4168-bf84-dc36241e80d4] succeeded in 2.1075826910091564s: None
[2025-01-20 12:00:00,060: INFO/MainProcess] Task leads.tasks.check_notifications[b4ab92c7-a34c-4a39-93c5-b957b3274244] received
[2025-01-20 12:00:02,258: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[b4ab92c7-a34c-4a39-93c5-b957b3274244] succeeded in 2.193795972969383s: None
[2025-01-21 12:00:00,096: INFO/MainProcess] Task leads.tasks.check_notifications[e5a68052-53d6-4726-827d-2daf57f4d96a] received
[2025-01-21 12:00:02,502: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[e5a68052-53d6-4726-827d-2daf57f4d96a] succeeded in 2.3992792210774496s: None
[2025-01-22 12:00:00,040: INFO/MainProcess] Task leads.tasks.check_notifications[536f1c9d-5157-451f-8955-95a51f49b599] received
[2025-01-22 12:00:02,154: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[536f1c9d-5157-451f-8955-95a51f49b599] succeeded in 2.10922004992608s: None
[2025-01-23 12:00:00,077: INFO/MainProcess] Task leads.tasks.check_notifications[23afa149-9f6e-45ce-8e98-59d992ba70a8] received
[2025-01-23 12:00:02,412: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[23afa149-9f6e-45ce-8e98-59d992ba70a8] succeeded in 2.3280310599366203s: None
[2025-01-24 12:00:00,079: INFO/MainProcess] Task leads.tasks.check_notifications[6cb93ad1-36f5-4e21-943d-84fcd1cad116] received
[2025-01-24 12:00:02,628: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[6cb93ad1-36f5-4e21-943d-84fcd1cad116] succeeded in 2.541180776897818s: None
[2025-01-25 12:00:00,098: INFO/MainProcess] Task leads.tasks.check_notifications[579ee942-20a8-4dcc-b4ff-eafb4e76bfa1] received
[2025-01-25 12:00:03,362: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[579ee942-20a8-4dcc-b4ff-eafb4e76bfa1] succeeded in 3.257101077120751s: None
[2025-01-26 12:00:00,080: INFO/MainProcess] Task leads.tasks.check_notifications[73a6a8b5-4d9c-4072-b4f4-fb543d17225c] received
[2025-01-26 12:00:02,488: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[73a6a8b5-4d9c-4072-b4f4-fb543d17225c] succeeded in 2.401027660118416s: None
[2025-01-27 12:00:00,110: INFO/MainProcess] Task leads.tasks.check_notifications[845570a6-243d-4193-b28d-641f7941fb43] received
[2025-01-27 12:00:02,844: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[845570a6-243d-4193-b28d-641f7941fb43] succeeded in 2.727487409953028s: None
[2025-01-28 12:00:00,031: INFO/MainProcess] Task leads.tasks.check_notifications[0f9c64a9-f37e-47a6-934f-79201bda675f] received
[2025-01-28 12:00:02,397: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[0f9c64a9-f37e-47a6-934f-79201bda675f] succeeded in 2.3596231159754097s: None
[2025-01-29 12:00:00,110: INFO/MainProcess] Task leads.tasks.check_notifications[58cdeda8-8606-41ef-a6b9-ebc8c75255a8] received
[2025-01-29 12:00:02,529: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[58cdeda8-8606-41ef-a6b9-ebc8c75255a8] succeeded in 2.412870013853535s: None
[2025-01-30 12:00:00,108: INFO/MainProcess] Task leads.tasks.check_notifications[5d4ba9ad-f2ce-408c-aed2-8274ed119313] received
[2025-01-30 12:00:02,675: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5d4ba9ad-f2ce-408c-aed2-8274ed119313] succeeded in 2.562533272895962s: None
[2025-01-31 12:00:00,066: INFO/MainProcess] Task leads.tasks.check_notifications[6f572553-4790-4fee-8a12-16a54667f95c] received
[2025-01-31 12:00:03,308: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[6f572553-4790-4fee-8a12-16a54667f95c] succeeded in 3.234396342886612s: None
[2025-02-01 12:00:00,092: INFO/MainProcess] Task leads.tasks.check_notifications[47e296cf-59a2-421f-ad00-988eb5138a70] received
[2025-02-01 12:00:02,968: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[47e296cf-59a2-421f-ad00-988eb5138a70] succeeded in 2.8711627039592713s: None
[2025-02-02 12:00:00,094: INFO/MainProcess] Task leads.tasks.check_notifications[6b57a016-acd4-43ad-8cc3-133fbda0f8df] received
[2025-02-02 12:00:02,536: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[6b57a016-acd4-43ad-8cc3-133fbda0f8df] succeeded in 2.437411095947027s: None
[2025-02-03 12:00:00,050: INFO/MainProcess] Task leads.tasks.check_notifications[26f38744-7b5f-4b19-97f3-3f15c3928044] received
[2025-02-03 12:00:02,532: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[26f38744-7b5f-4b19-97f3-3f15c3928044] succeeded in 2.4774867959786206s: None
[2025-02-04 12:00:00,106: INFO/MainProcess] Task leads.tasks.check_notifications[2b4cb62a-c721-4c7c-be5c-a98598fbb92c] received
[2025-02-04 12:00:02,722: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[2b4cb62a-c721-4c7c-be5c-a98598fbb92c] succeeded in 2.609411641024053s: None
[2025-02-05 12:00:00,088: INFO/MainProcess] Task leads.tasks.check_notifications[7f450b3f-55d5-4fa4-8946-4b7cb655df76] received
[2025-02-05 12:00:02,726: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[7f450b3f-55d5-4fa4-8946-4b7cb655df76] succeeded in 2.6337272189557552s: None
[2025-02-06 12:00:00,107: INFO/MainProcess] Task leads.tasks.check_notifications[9d9a3234-84a8-4ccf-b547-5c81d53951d8] received
[2025-02-06 12:00:02,544: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[9d9a3234-84a8-4ccf-b547-5c81d53951d8] succeeded in 2.4321020748466253s: None
[2025-02-07 12:00:00,070: INFO/MainProcess] Task leads.tasks.check_notifications[dd5577ad-4cc5-4e26-b4b1-d89640b03b12] received
[2025-02-07 12:00:02,640: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[dd5577ad-4cc5-4e26-b4b1-d89640b03b12] succeeded in 2.5646141348406672s: None
[2025-02-08 12:00:00,054: INFO/MainProcess] Task leads.tasks.check_notifications[6df7e707-c0a1-4bea-a752-1e6c6bbae14b] received
[2025-02-08 12:00:02,730: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[6df7e707-c0a1-4bea-a752-1e6c6bbae14b] succeeded in 2.6722009922377765s: None
[2025-02-09 12:00:00,047: INFO/MainProcess] Task leads.tasks.check_notifications[db88c629-8bb4-4e63-b7bb-660873eaf0d0] received
[2025-02-09 12:00:02,403: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[db88c629-8bb4-4e63-b7bb-660873eaf0d0] succeeded in 2.351044640876353s: None
[2025-02-10 12:00:00,112: INFO/MainProcess] Task leads.tasks.check_notifications[1b99de70-6234-48d9-9a12-9a0ad6ba9f96] received
[2025-02-10 12:00:02,671: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[1b99de70-6234-48d9-9a12-9a0ad6ba9f96] succeeded in 2.5538649656809866s: None
[2025-02-11 12:00:00,033: INFO/MainProcess] Task leads.tasks.check_notifications[8d42cee0-b557-401d-86d0-4002913fbb4f] received
[2025-02-11 12:00:02,453: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[8d42cee0-b557-401d-86d0-4002913fbb4f] succeeded in 2.4155291030183434s: None
[2025-02-12 12:00:00,108: INFO/MainProcess] Task leads.tasks.check_notifications[8e11e49f-3783-4113-b93f-1e15dd21a5c9] received
[2025-02-12 12:00:02,519: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[8e11e49f-3783-4113-b93f-1e15dd21a5c9] succeeded in 2.4048953391611576s: None
[2025-02-13 12:00:00,103: INFO/MainProcess] Task leads.tasks.check_notifications[84cfe06a-eebb-4436-94c9-9483d474e4a3] received
[2025-02-13 12:00:02,775: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[84cfe06a-eebb-4436-94c9-9483d474e4a3] succeeded in 2.668170069810003s: None
[2025-02-14 12:00:00,076: INFO/MainProcess] Task leads.tasks.check_notifications[29dbda0d-3c00-41c0-8152-6260e6d3b461] received
[2025-02-14 12:00:02,784: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[29dbda0d-3c00-41c0-8152-6260e6d3b461] succeeded in 2.703462954144925s: None
[2025-02-15 12:00:00,101: INFO/MainProcess] Task leads.tasks.check_notifications[c6f3e762-be3c-40ae-ba83-27b28ec9744c] received
[2025-02-15 12:00:02,574: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c6f3e762-be3c-40ae-ba83-27b28ec9744c] succeeded in 2.4683342068456113s: None
[2025-02-16 12:00:00,099: INFO/MainProcess] Task leads.tasks.check_notifications[2766f26e-5526-499c-9763-8ce7584642e3] received
[2025-02-16 12:00:02,461: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[2766f26e-5526-499c-9763-8ce7584642e3] succeeded in 2.357943271752447s: None
[2025-02-17 12:00:00,027: INFO/MainProcess] Task leads.tasks.check_notifications[04b6df25-4b28-41b5-a3a4-50e6f16f3160] received
[2025-02-17 12:00:02,929: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[04b6df25-4b28-41b5-a3a4-50e6f16f3160] succeeded in 2.8971411329694092s: None
[2025-02-18 12:00:00,072: INFO/MainProcess] Task leads.tasks.check_notifications[c44be5be-5c57-4f74-825f-a3eac6130c3f] received
[2025-02-18 12:00:02,779: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c44be5be-5c57-4f74-825f-a3eac6130c3f] succeeded in 2.701030833646655s: None
[2025-02-19 12:00:00,075: INFO/MainProcess] Task leads.tasks.check_notifications[13edd200-33fc-4759-990e-3bc6092487f6] received
[2025-02-19 12:00:02,288: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[13edd200-33fc-4759-990e-3bc6092487f6] succeeded in 2.2083591860719025s: None
[2025-02-20 12:00:00,058: INFO/MainProcess] Task leads.tasks.check_notifications[5c54ce29-d143-49f1-a247-6cbb16f148da] received
[2025-02-20 12:00:02,679: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5c54ce29-d143-49f1-a247-6cbb16f148da] succeeded in 2.6162011618725955s: None
[2025-02-21 12:00:00,104: INFO/MainProcess] Task leads.tasks.check_notifications[d82bb0e8-5ef1-49c5-841d-c2465bac61ba] received
[2025-02-21 12:00:02,330: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[d82bb0e8-5ef1-49c5-841d-c2465bac61ba] succeeded in 2.2216380396857858s: None
[2025-02-22 12:00:00,097: INFO/MainProcess] Task leads.tasks.check_notifications[efd9894d-861c-44dc-83d6-2638657500c2] received
[2025-02-22 12:00:02,762: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[efd9894d-861c-44dc-83d6-2638657500c2] succeeded in 2.660276287700981s: None
[2025-02-22 14:21:45,850: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 340, in start
    blueprint.start(self)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/bootsteps.py", line 116, in start
    step.start(parent)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 746, in start
    c.loop(*c.loop_args())
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/loops.py", line 97, in asynloop
    next(loop)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/asynchronous/hub.py", line 373, in create_loop
    cb(*cbargs)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 1352, in on_readable
    self.cycle.on_readable(fileno)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 569, in on_readable
    chan.handlers[type]()
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 918, in _receive
    ret.append(self._receive_one(c))
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 928, in _receive_one
    response = c.parse_response()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 865, in parse_response
    response = self._execute(conn, try_read)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 841, in _execute
    return conn.retry.call_with_retry(
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 65, in call_with_retry
    fail(error)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 843, in <lambda>
    lambda error: self._disconnect_raise_connect(conn, error),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 830, in _disconnect_raise_connect
    raise error
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 62, in call_with_retry
    return do()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 842, in <lambda>
    lambda: command(*args, **kwargs),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 863, in try_read
    return conn.read_response(disconnect_on_error=False, push_request=True)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/connection.py", line 592, in read_response
    response = self._parser.read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 15, in read_response
    result = self._read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 25, in _read_response
    raw = self._buffer.readline()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 115, in readline
    self._read_from_socket()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 68, in _read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
[2025-02-22 14:21:46,054: WARNING/MainProcess] /home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:391: CPendingDeprecationWarning: 
In Celery 5.1 we introduced an optional breaking change which
on connection loss cancels all currently executed tasks with late acknowledgement enabled.
These tasks cannot be acknowledged as the connection is gone, and the tasks are automatically redelivered
back to the queue. You can enable this behavior using the worker_cancel_long_running_tasks_on_connection_loss
setting. In Celery 5.1 it is set to False by default. The setting will be set to True by default in Celery 6.0.

  warnings.warn(CANCEL_TASKS_BY_DEFAULT, CPendingDeprecationWarning)

[2025-02-22 14:21:46,245: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 2.00 seconds... (1/100)

[2025-02-22 14:21:48,257: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 4.00 seconds... (2/100)


worker: Warm shutdown (MainProcess)
 
 -------------- celery@ubuntu v5.4.0 (opalescent)
--- ***** ----- 
-- ******* ---- Linux-5.15.0-131-generic-x86_64-with-glibc2.35 2025-02-22 14:22:37
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         new_tdl_site:0x7f354cc54e50
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 1 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . leads.tasks.check_notifications

[2025-02-22 14:22:38,008: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-02-22 14:22:38,013: INFO/MainProcess] mingle: searching for neighbors
[2025-02-22 14:22:39,028: INFO/MainProcess] mingle: all alone
[2025-02-22 14:22:39,048: INFO/MainProcess] celery@ubuntu ready.
[2025-02-23 12:00:00,080: INFO/MainProcess] Task leads.tasks.check_notifications[3cb4d006-94b9-4b08-8e73-0bafbd8da805] received
[2025-02-23 12:00:02,605: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[3cb4d006-94b9-4b08-8e73-0bafbd8da805] succeeded in 2.520087792989216s: None
[2025-02-24 12:00:00,073: INFO/MainProcess] Task leads.tasks.check_notifications[de08f68e-a5d5-4590-9390-581be86a81db] received
[2025-02-24 12:00:02,819: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[de08f68e-a5d5-4590-9390-581be86a81db] succeeded in 2.7429605860088486s: None
[2025-02-25 12:00:00,046: INFO/MainProcess] Task leads.tasks.check_notifications[8a060be8-0623-438a-ba4e-c934c18a7485] received
[2025-02-25 12:00:02,652: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[8a060be8-0623-438a-ba4e-c934c18a7485] succeeded in 2.6027929229894653s: None
[2025-02-26 12:00:00,103: INFO/MainProcess] Task leads.tasks.check_notifications[3a7429ad-3801-40a7-9f0a-80bf7a548b28] received
[2025-02-26 12:00:02,453: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[3a7429ad-3801-40a7-9f0a-80bf7a548b28] succeeded in 2.3451927170390263s: None
[2025-02-27 12:00:00,086: INFO/MainProcess] Task leads.tasks.check_notifications[eded339b-eb38-438f-905e-25a1334a5b86] received
[2025-02-27 12:00:02,644: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[eded339b-eb38-438f-905e-25a1334a5b86] succeeded in 2.553911696013529s: None
[2025-02-28 12:00:00,102: INFO/MainProcess] Task leads.tasks.check_notifications[5d10a794-2e8c-4e46-9049-494f535d8afe] received
[2025-02-28 12:00:02,569: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5d10a794-2e8c-4e46-9049-494f535d8afe] succeeded in 2.4630088940029964s: None
[2025-03-01 12:00:00,062: INFO/MainProcess] Task leads.tasks.check_notifications[0cb660c1-34fd-4f71-a159-93d2b1a5c846] received
[2025-03-01 12:00:02,667: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[0cb660c1-34fd-4f71-a159-93d2b1a5c846] succeeded in 2.601300850044936s: None
[2025-03-02 12:00:00,100: INFO/MainProcess] Task leads.tasks.check_notifications[aac82dc0-862c-4052-9394-66017d883ce5] received
[2025-03-02 12:00:02,664: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[aac82dc0-862c-4052-9394-66017d883ce5] succeeded in 2.559656225028448s: None
[2025-03-03 12:00:00,083: INFO/MainProcess] Task leads.tasks.check_notifications[c9156067-4b26-448f-a961-0f7061604106] received
[2025-03-03 12:00:02,830: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c9156067-4b26-448f-a961-0f7061604106] succeeded in 2.7440030720317736s: None
[2025-03-04 12:00:00,106: INFO/MainProcess] Task leads.tasks.check_notifications[717be2d6-51ab-4d9f-b658-ecac6197b02e] received
[2025-03-04 12:00:02,726: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[717be2d6-51ab-4d9f-b658-ecac6197b02e] succeeded in 2.615653568995185s: None
[2025-03-04 22:45:07,901: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 340, in start
    blueprint.start(self)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/bootsteps.py", line 116, in start
    step.start(parent)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 746, in start
    c.loop(*c.loop_args())
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/loops.py", line 97, in asynloop
    next(loop)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/asynchronous/hub.py", line 373, in create_loop
    cb(*cbargs)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 1352, in on_readable
    self.cycle.on_readable(fileno)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 569, in on_readable
    chan.handlers[type]()
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 918, in _receive
    ret.append(self._receive_one(c))
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 928, in _receive_one
    response = c.parse_response()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 865, in parse_response
    response = self._execute(conn, try_read)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 841, in _execute
    return conn.retry.call_with_retry(
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 65, in call_with_retry
    fail(error)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 843, in <lambda>
    lambda error: self._disconnect_raise_connect(conn, error),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 830, in _disconnect_raise_connect
    raise error
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 62, in call_with_retry
    return do()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 842, in <lambda>
    lambda: command(*args, **kwargs),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 863, in try_read
    return conn.read_response(disconnect_on_error=False, push_request=True)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/connection.py", line 592, in read_response
    response = self._parser.read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 15, in read_response
    result = self._read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 25, in _read_response
    raw = self._buffer.readline()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 115, in readline
    self._read_from_socket()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 68, in _read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
[2025-03-04 22:45:08,094: WARNING/MainProcess] /home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:391: CPendingDeprecationWarning: 
In Celery 5.1 we introduced an optional breaking change which
on connection loss cancels all currently executed tasks with late acknowledgement enabled.
These tasks cannot be acknowledged as the connection is gone, and the tasks are automatically redelivered
back to the queue. You can enable this behavior using the worker_cancel_long_running_tasks_on_connection_loss
setting. In Celery 5.1 it is set to False by default. The setting will be set to True by default in Celery 6.0.

  warnings.warn(CANCEL_TASKS_BY_DEFAULT, CPendingDeprecationWarning)

[2025-03-04 22:45:08,248: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 2.00 seconds... (1/100)

[2025-03-04 22:45:10,289: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 4.00 seconds... (2/100)


worker: Warm shutdown (MainProcess)
 
 -------------- celery@ubuntu v5.4.0 (opalescent)
--- ***** ----- 
-- ******* ---- Linux-5.15.0-133-generic-x86_64-with-glibc2.35 2025-03-04 22:46:04
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         new_tdl_site:0x7f285b1c8e50
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 1 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . leads.tasks.check_notifications

[2025-03-04 22:46:04,440: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-03-04 22:46:04,446: INFO/MainProcess] mingle: searching for neighbors
[2025-03-04 22:46:05,462: INFO/MainProcess] mingle: all alone
[2025-03-04 22:46:05,480: INFO/MainProcess] celery@ubuntu ready.
[2025-03-05 12:00:00,129: INFO/MainProcess] Task leads.tasks.check_notifications[c2423693-9bc8-4304-b8fb-1614fcf105dc] received
[2025-03-05 12:00:02,857: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c2423693-9bc8-4304-b8fb-1614fcf105dc] succeeded in 2.723249101996771s: None
[2025-03-06 12:00:00,106: INFO/MainProcess] Task leads.tasks.check_notifications[1027b5ff-d55c-4b8a-bddf-97ec1db17047] received
[2025-03-06 12:00:02,890: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[1027b5ff-d55c-4b8a-bddf-97ec1db17047] succeeded in 2.7792404819920193s: None
[2025-03-07 12:00:00,106: INFO/MainProcess] Task leads.tasks.check_notifications[a8b2403a-1ab4-4c9d-bb8b-e482dff80bdc] received
[2025-03-07 12:00:02,977: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[a8b2403a-1ab4-4c9d-bb8b-e482dff80bdc] succeeded in 2.8681422399822623s: None
[2025-03-08 12:00:00,103: INFO/MainProcess] Task leads.tasks.check_notifications[d79f2e42-5821-4fc2-b0d1-d4c39a19bebd] received
[2025-03-08 12:00:03,095: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[d79f2e42-5821-4fc2-b0d1-d4c39a19bebd] succeeded in 2.989863829978276s: None
[2025-03-09 12:00:00,072: INFO/MainProcess] Task leads.tasks.check_notifications[f1e384ca-61fc-4677-848d-070bd627be8c] received
[2025-03-09 12:00:02,980: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[f1e384ca-61fc-4677-848d-070bd627be8c] succeeded in 2.904924397007562s: None
[2025-03-10 12:00:00,100: INFO/MainProcess] Task leads.tasks.check_notifications[5550c559-9d17-4c70-9d1f-4ac0f04aa972] received
[2025-03-10 12:00:02,818: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5550c559-9d17-4c70-9d1f-4ac0f04aa972] succeeded in 2.713846334023401s: None
[2025-03-11 12:00:00,066: INFO/MainProcess] Task leads.tasks.check_notifications[56cb3873-b381-4ed1-9dca-2eb25ed07dc0] received
[2025-03-11 12:00:02,343: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[56cb3873-b381-4ed1-9dca-2eb25ed07dc0] succeeded in 2.273939032922499s: None
[2025-03-12 12:00:00,039: INFO/MainProcess] Task leads.tasks.check_notifications[139b256a-c124-4261-a597-a5c060f5274e] received
[2025-03-12 12:00:02,794: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[139b256a-c124-4261-a597-a5c060f5274e] succeeded in 2.7488449959782884s: None
[2025-03-13 12:00:00,104: INFO/MainProcess] Task leads.tasks.check_notifications[0a79b565-9b8f-4108-8a16-51471fc1c6b1] received
[2025-03-13 12:00:03,399: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[0a79b565-9b8f-4108-8a16-51471fc1c6b1] succeeded in 3.2914998800260946s: None
[2025-03-14 12:00:00,104: INFO/MainProcess] Task leads.tasks.check_notifications[3250562c-213a-4c3a-bac6-62faa540ba28] received
[2025-03-14 12:00:02,413: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[3250562c-213a-4c3a-bac6-62faa540ba28] succeeded in 2.304169106995687s: None
[2025-03-15 12:00:00,102: INFO/MainProcess] Task leads.tasks.check_notifications[5787686c-b01c-4614-a0ee-472c7f76c9de] received
[2025-03-15 12:00:04,544: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5787686c-b01c-4614-a0ee-472c7f76c9de] succeeded in 4.438920204993337s: None
[2025-03-16 12:00:00,081: INFO/MainProcess] Task leads.tasks.check_notifications[c386707e-f87e-448b-92bb-25303f813d15] received
[2025-03-16 12:00:02,424: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c386707e-f87e-448b-92bb-25303f813d15] succeeded in 2.338232500012964s: None
[2025-03-17 12:00:00,061: INFO/MainProcess] Task leads.tasks.check_notifications[c74d8fd6-c622-4602-85bd-e5feef33ac0a] received
[2025-03-17 12:00:03,099: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[c74d8fd6-c622-4602-85bd-e5feef33ac0a] succeeded in 3.0335951279848814s: None
[2025-03-18 12:00:00,085: INFO/MainProcess] Task leads.tasks.check_notifications[28dd0198-8e44-4276-a34b-0494d3c1b4cb] received
[2025-03-18 12:00:02,616: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[28dd0198-8e44-4276-a34b-0494d3c1b4cb] succeeded in 2.52820763993077s: None
[2025-03-18 23:55:32,014: WARNING/MainProcess] consumer: Connection to broker lost. Trying to re-establish the connection...
Traceback (most recent call last):
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 340, in start
    blueprint.start(self)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/bootsteps.py", line 116, in start
    step.start(parent)
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py", line 746, in start
    c.loop(*c.loop_args())
  File "/home/igor/venv/lib/python3.10/site-packages/celery/worker/loops.py", line 97, in asynloop
    next(loop)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/asynchronous/hub.py", line 373, in create_loop
    cb(*cbargs)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 1352, in on_readable
    self.cycle.on_readable(fileno)
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 569, in on_readable
    chan.handlers[type]()
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 918, in _receive
    ret.append(self._receive_one(c))
  File "/home/igor/venv/lib/python3.10/site-packages/kombu/transport/redis.py", line 928, in _receive_one
    response = c.parse_response()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 865, in parse_response
    response = self._execute(conn, try_read)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 841, in _execute
    return conn.retry.call_with_retry(
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 65, in call_with_retry
    fail(error)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 843, in <lambda>
    lambda error: self._disconnect_raise_connect(conn, error),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 830, in _disconnect_raise_connect
    raise error
  File "/home/igor/venv/lib/python3.10/site-packages/redis/retry.py", line 62, in call_with_retry
    return do()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 842, in <lambda>
    lambda: command(*args, **kwargs),
  File "/home/igor/venv/lib/python3.10/site-packages/redis/client.py", line 863, in try_read
    return conn.read_response(disconnect_on_error=False, push_request=True)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/connection.py", line 592, in read_response
    response = self._parser.read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 15, in read_response
    result = self._read_response(disable_decoding=disable_decoding)
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/resp2.py", line 25, in _read_response
    raw = self._buffer.readline()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 115, in readline
    self._read_from_socket()
  File "/home/igor/venv/lib/python3.10/site-packages/redis/_parsers/socket.py", line 68, in _read_from_socket
    raise ConnectionError(SERVER_CLOSED_CONNECTION_ERROR)
redis.exceptions.ConnectionError: Connection closed by server.
[2025-03-18 23:55:32,192: WARNING/MainProcess] /home/igor/venv/lib/python3.10/site-packages/celery/worker/consumer/consumer.py:391: CPendingDeprecationWarning: 
In Celery 5.1 we introduced an optional breaking change which
on connection loss cancels all currently executed tasks with late acknowledgement enabled.
These tasks cannot be acknowledged as the connection is gone, and the tasks are automatically redelivered
back to the queue. You can enable this behavior using the worker_cancel_long_running_tasks_on_connection_loss
setting. In Celery 5.1 it is set to False by default. The setting will be set to True by default in Celery 6.0.

  warnings.warn(CANCEL_TASKS_BY_DEFAULT, CPendingDeprecationWarning)

[2025-03-18 23:55:32,254: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 2.00 seconds... (1/100)

[2025-03-18 23:55:34,290: ERROR/MainProcess] consumer: Cannot connect to redis://localhost:6379/0: Error 111 connecting to localhost:6379. Connection refused..
Trying again in 4.00 seconds... (2/100)


worker: Warm shutdown (MainProcess)
 
 -------------- celery@ubuntu v5.4.0 (opalescent)
--- ***** ----- 
-- ******* ---- Linux-5.15.0-134-generic-x86_64-with-glibc2.35 2025-03-18 23:56:27
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         new_tdl_site:0x7f61c97e0e50
- ** ---------- .> transport:   redis://localhost:6379/0
- ** ---------- .> results:     redis://localhost:6379/0
- *** --- * --- .> concurrency: 1 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> celery           exchange=celery(direct) key=celery
                

[tasks]
  . leads.tasks.check_notifications

[2025-03-18 23:56:28,070: INFO/MainProcess] Connected to redis://localhost:6379/0
[2025-03-18 23:56:28,075: INFO/MainProcess] mingle: searching for neighbors
[2025-03-18 23:56:29,091: INFO/MainProcess] mingle: all alone
[2025-03-18 23:56:29,109: INFO/MainProcess] celery@ubuntu ready.
[2025-03-19 12:00:00,123: INFO/MainProcess] Task leads.tasks.check_notifications[db6b6237-23f0-463d-a5b8-038368d0fd59] received
[2025-03-19 12:00:02,372: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[db6b6237-23f0-463d-a5b8-038368d0fd59] succeeded in 2.245043759001419s: None
[2025-03-20 12:00:00,057: INFO/MainProcess] Task leads.tasks.check_notifications[0ba691ba-89e2-4ea3-bd13-f582ed8a39b6] received
[2025-03-20 12:00:02,697: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[0ba691ba-89e2-4ea3-bd13-f582ed8a39b6] succeeded in 2.6337608379981248s: None
[2025-03-21 12:00:00,080: INFO/MainProcess] Task leads.tasks.check_notifications[1b96ae14-d163-43ba-b322-913381b1ddd7] received
[2025-03-21 12:00:02,544: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[1b96ae14-d163-43ba-b322-913381b1ddd7] succeeded in 2.4574479580041952s: None
[2025-03-22 12:00:00,105: INFO/MainProcess] Task leads.tasks.check_notifications[e8d92c0b-832a-461e-96f5-7b014384da87] received
[2025-03-22 12:00:03,427: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[e8d92c0b-832a-461e-96f5-7b014384da87] succeeded in 3.316074568021577s: None
[2025-03-23 12:00:00,098: INFO/MainProcess] Task leads.tasks.check_notifications[5e5612d8-e7ce-4296-8583-d32c5e7e12d5] received
[2025-03-23 12:00:02,681: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5e5612d8-e7ce-4296-8583-d32c5e7e12d5] succeeded in 2.578852917999029s: None
[2025-03-24 12:00:00,044: INFO/MainProcess] Task leads.tasks.check_notifications[04cf1b5d-48c9-4bed-bf7d-b406714d6aaf] received
[2025-03-24 12:00:02,778: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[04cf1b5d-48c9-4bed-bf7d-b406714d6aaf] succeeded in 2.7282966570346616s: None
[2025-03-25 12:00:00,039: INFO/MainProcess] Task leads.tasks.check_notifications[f1c49eeb-e24e-4960-b282-dcfcf715c330] received
[2025-03-25 12:00:02,701: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[f1c49eeb-e24e-4960-b282-dcfcf715c330] succeeded in 2.6564355270238593s: None
[2025-03-26 12:00:00,112: INFO/MainProcess] Task leads.tasks.check_notifications[2b7be50d-e3bb-4a63-9280-24c5aab9445e] received
[2025-03-26 12:00:02,602: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[2b7be50d-e3bb-4a63-9280-24c5aab9445e] succeeded in 2.4823222160339355s: None
[2025-03-27 12:00:00,068: INFO/MainProcess] Task leads.tasks.check_notifications[3b1866a0-d977-4dd4-afff-6c60df51a385] received
[2025-03-27 12:00:02,634: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[3b1866a0-d977-4dd4-afff-6c60df51a385] succeeded in 2.561387502006255s: None
[2025-03-28 12:00:00,103: INFO/MainProcess] Task leads.tasks.check_notifications[5677f137-2438-4497-a135-bd0412c1441e] received
[2025-03-28 12:00:02,382: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[5677f137-2438-4497-a135-bd0412c1441e] succeeded in 2.2747501040576026s: None
[2025-03-29 12:00:00,079: INFO/MainProcess] Task leads.tasks.check_notifications[d729e587-3bde-491f-ae40-dfa95772a0a4] received
[2025-03-29 12:00:02,314: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[d729e587-3bde-491f-ae40-dfa95772a0a4] succeeded in 2.228910726029426s: None
[2025-03-30 12:00:00,078: INFO/MainProcess] Task leads.tasks.check_notifications[58b1fd80-4cf2-498f-8e9f-109ef3a89d5c] received
[2025-03-30 12:00:03,924: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[58b1fd80-4cf2-498f-8e9f-109ef3a89d5c] succeeded in 3.841917929938063s: None
[2025-03-31 12:00:00,087: INFO/MainProcess] Task leads.tasks.check_notifications[536916d1-2cbb-4da6-8ffd-de04c754004a] received
[2025-03-31 12:00:02,544: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[536916d1-2cbb-4da6-8ffd-de04c754004a] succeeded in 2.4523348610382527s: None
[2025-04-01 12:00:00,074: INFO/MainProcess] Task leads.tasks.check_notifications[dad0bad2-9232-4343-8180-9c083dfb5a88] received
[2025-04-01 12:00:02,703: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[dad0bad2-9232-4343-8180-9c083dfb5a88] succeeded in 2.6233478400390595s: None
[2025-04-02 12:00:00,067: INFO/MainProcess] Task leads.tasks.check_notifications[e324f77d-ec96-4f4a-a499-473ee3ddafe7] received
[2025-04-02 12:00:02,934: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[e324f77d-ec96-4f4a-a499-473ee3ddafe7] succeeded in 2.863431043922901s: None
[2025-04-03 12:00:00,099: INFO/MainProcess] Task leads.tasks.check_notifications[7656edbb-7769-458d-be42-96f60024614b] received
[2025-04-03 12:00:02,777: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[7656edbb-7769-458d-be42-96f60024614b] succeeded in 2.6750697130337358s: None
[2025-04-04 12:00:00,104: INFO/MainProcess] Task leads.tasks.check_notifications[db1cccc1-62e5-470b-a784-92b5abc5107f] received
[2025-04-04 12:00:02,867: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[db1cccc1-62e5-470b-a784-92b5abc5107f] succeeded in 2.757524851942435s: None
[2025-04-05 12:00:00,096: INFO/MainProcess] Task leads.tasks.check_notifications[23c25b9c-59a0-4fe1-b849-9997aef03617] received
[2025-04-05 12:00:02,404: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[23c25b9c-59a0-4fe1-b849-9997aef03617] succeeded in 2.3039730321615934s: None
[2025-04-06 12:00:00,089: INFO/MainProcess] Task leads.tasks.check_notifications[24c8303b-8e00-463a-aba9-d65e064ee31e] received
[2025-04-06 12:00:03,066: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[24c8303b-8e00-463a-aba9-d65e064ee31e] succeeded in 2.9721283849794418s: None
[2025-04-07 12:00:00,075: INFO/MainProcess] Task leads.tasks.check_notifications[d68219ad-8ee4-4146-b3d1-3464fcc988d6] received
[2025-04-07 12:00:02,868: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[d68219ad-8ee4-4146-b3d1-3464fcc988d6] succeeded in 2.787250906927511s: None
[2025-04-08 12:00:00,081: INFO/MainProcess] Task leads.tasks.check_notifications[acb16a1c-622a-479a-9b17-37d065a0e6d4] received
[2025-04-08 12:00:02,339: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[acb16a1c-622a-479a-9b17-37d065a0e6d4] succeeded in 2.253500512102619s: None
[2025-04-09 12:00:00,048: INFO/MainProcess] Task leads.tasks.check_notifications[e30f2de7-5719-428f-8cd2-e06cce33990c] received
[2025-04-09 12:00:02,705: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[e30f2de7-5719-428f-8cd2-e06cce33990c] succeeded in 2.6533049570862204s: None
[2025-04-10 12:00:00,053: INFO/MainProcess] Task leads.tasks.check_notifications[1564e499-45b3-4195-ad77-9922210b09f4] received
[2025-04-10 12:00:03,277: INFO/ForkPoolWorker-1] Task leads.tasks.check_notifications[1564e499-45b3-4195-ad77-9922210b09f4] succeeded in 3.218933332944289s: None
